frontend/views/pages/Contributions.vue

```diff
@@ -2,167 +2,165 @@
 
-  page-section(title='Debug Income Details')
-    form(
-      novalidate
-      @submit.prevent='setPaymentInfo'
-    )
-      // TODO: change this so that the input *inside* of the label
-      //       as Sandrina describes here (using a fieldset):
-      //       https://github.com/okTurtles/group-income-simple/pull/705#discussion_r335817744
-      .field
-        i18n.label(
-          tag='label'
-          :args='{ groupMincomeFormatted }'
-        ) Do you make at least {groupMincomeFormatted} per month?
+        i18n.has-text-1.spacer-around(
+          v-else-if='needsIncome && !hasPayments'
+          tag='p'
+        ) No one is pledging money at the moment.
 
-        .radio-wrapper
-          input.radio(
-            type='radio'
-            value='incomeAmount'
-            v-model='form.incomeDetailsType'
-            id='radio-receiving'
+        ul.spacer(
+          v-if='doesReceive'
+          data-test='revceivingList'
+        )
+          contribution(
+            v-if='needsIncome && hasPayments'
           )
-          //- TODO: update design system to make radio buttons use labels instead of spans (and update _forms.scss accordingly)
-          i18n(tag='label' for='radio-receiving') No, I don't
+            contribution-item(
+              :what='receivingMonetary.total'
+              :who='receivingMonetary.list'
+              type='MONETARY'
+            )
 
-        .radio-wrapper
-          input.radio(
-            type='radio'
-            value='pledgeAmount'
-            v-model='form.incomeDetailsType'
-            id='radio-pleding'
+          contribution(
+            v-for='(contribution, index) in receivingNonMonetary'
+            :key='`contribution-${index}`'
           )
-          i18n(tag='label' for='radio-pleding') Yes, I do
+            contribution-item(
+              :what='contribution.what'
+              :who='contribution.who'
+              type='NON_MONETARY'
+            )
 
-      //- NOTE: this "key='formReceiving'" is needed or else the v-if doesn't work
-      //- for some reason...
-      .field(v-if='form.incomeDetailsType === "incomeAmount"' key='formReceiving')
-        i18n.label(tag='label') Enter your income:
-        input.input.is-primary(
-          type='text'
-          v-model='$v.form.incomeAmount.$model'
-          :class='{error: $v.form.incomeAmount.$error}'
-          v-error:incomeAmount='{ tag: "p", attrs: { "data-test": "badIncome" } }'
+        button.button.is-small(
+          v-if='groupMembersCount === 0'
+          @click='openModal("InviteByLink")'
         )
+          i.icon-plus
+          i18n Add members to group
+
+      .giving
+        i18n(tag='h3' class='card-header') Giving
+
+        i18n.has-text-1.spacer-around(
+          v-if='!doesGive'
+          tag='p'
+        ) You can contribute to your group with money or other valuables like teaching skills, sharing your time ot help someone. The sky is the limit!
```

Comment:
Here the logic is broken, resulting in Problem 3. doesGive needs to be:

```js
    doesGive () {
      return this.hasPayments || this.memberGroupProfile.nonMonetaryContributions.length > 0
    },
```

Note that we also need to make sure to show the `Add a non-monetary pledge` button

Reply:
That's what I figure out this morning yes